src/styles/_article.scss

```diff
@@ -0,0 +1,61 @@
+#article {
+  header {
+    position: relative;
+    background: $finsec-royal-blue;
+  }
+  main {
+    h2,
+    h3,
+    h4,
+    p {
+      margin-top: 20px;
+    }
+    section {
+      align-items: left;
+      margin-bottom: 40px;
+      max-width: 1260px;
+      img {
+        max-width: 100%;
```

Comment:
so when the screen size reduces down, the max-width 100% keeps the image inside it's parens element as the screen width continues to reduce